features/post.feature

```diff
@@ -29,6 +29,28 @@ Feature: Manage WordPress posts
     When I try the previous command again
     Then the return code should be 1
 
+    When I run `wp term create category "First Category" --porcelain`
+    And save STDOUT as {TERM_ID}
+    And I run `wp term create category "Second Category" --porcelain`
+    And save STDOUT as {SECOND_TERM_ID}
+
+    When I run `wp post create --post_title="Test category" --post_category="First Category" --porcelain`
+    Then STDOUT should be a number
+    And save STDOUT as {POST_ID}
+
+    When I run `wp post update {POST_ID} --post_category={SECOND_TERM_ID}`
+    Then STDOUT should be:
+      """
+      Success: Updated post {POST_ID}.
+      """
+
+    When I run `wp post update {POST_ID} --post_category=test --porcelain`
```

Comment:
If a command produces a warning or an error, it appears on `STDERR`, and need to use `try` instead of `run`:
```
    When I try `wp post update {POST_ID} --post_category=test`
    Then STDERR should be:
      """
      Warning: Unable to set test as post category
      """
    And STDOUT should be:
      """
      Success: Updated post {POST_ID}.
      """
```

Reply:
(This is a real gotcha with the behat tests, and we should throw a reminder to use `try` when it happens as it's so easy to forget to do it...)